AGPs/AGP-Frame.md

```diff
@@ -0,0 +1,44 @@
+---
+AGP: ?
+Title: Flock Funding for Frame
+Author: Jordan Muir (@floating)
+Status: Pending
+Track: Finance
+Created: 2019/10/2
+---
+
+# AGP-Frame: Flock Funding for Frame
+## Address of the Transfer Recipient
+We request funds to be sent to our Aragon DAO: [`frame.aragonid.eth`](https://mainnet.aragon.org/#/frame.aragonid.eth) 0xe30b0d5168dfd8d92bc511e814de14e3757c775e
+
+## Amount of Transfer
+284,000 DAI
+
+50,000 ANT per year for the next 4 years with a 1 year cliff
+
+## Number of Transfers
+1
+
+## Purpose of Transfer
+Frame joining Flock allows us to **continue development and support of Aragon related initiatives**. We will continue **adding fundamental value to the Aragon ecosystem at the platform layer**. We'd like to embark on **deeper coordination within Aragon to provide support for any platform Aragon chooses to build on: desktop, mobile, Ethereum, Polkadot, IPFS, etc.** We’re looking forward to working with current and future Flock/Nest teams to fulfill their needs at this layer and to create a unified effort towards wider adoption. **We’ve made it our mission to keep our proposal focused and will over-deliver on our initiatives. We think our vision for adding value to Aragon is compelling and thoughtful. We hope others see the impact we'll have if included in Flock.**
+
+Frame being included in Flock gives Aragon the ability to create, iterate, improve and maintain a path for users to go from zero knowledge, to participating in a DAO. **This will be a very important feature for bigger teams, communities and organizations that want to start using Aragon. Large groups will be able to send out a simple link to all members, allowing everyone to seamlessly get up and running using their DAO without any external support.**
```

Comment:
```suggestion
Frame being included in Flock gives Aragon the ability to create, iterate, improve, and maintain a path for users to go from zero knowledge, to participating in a DAO. **This will be a very important feature for bigger teams, communities, and organizations that want to start using Aragon. Large groups will be able to send out a simple link to all members, allowing everyone to seamlessly get up and running using their DAO without any external support.**
```